test/ejabberd_tests/tests/amp_SUITE.erl

```diff
@@ -11,37 +11,82 @@
 -include_lib("escalus/include/escalus_xmlns.hrl").
 -include_lib("exml/include/exml.hrl").
 
-all() -> [{group, all}].
+-define(DOMAIN, <<"localhost">>).
```

Comment:
Domain should be read from configuration like this:
```erlang
ct:get_config({hosts, mim, domain}).
```
Hardcoding this may hurt sooner or later.